node/connection_base.js

```diff
@@ -66,7 +66,7 @@ function TChannelConnectionBase(channel, direction, socketRemoteAddr) {
         connection: self
     });
 
-    self.guid = ++CONNECTION_BASE_IDENTIFIER;
+    self.guid = String(++CONNECTION_BASE_IDENTIFIER + '~');
```

Comment:
++CONNECTION_BASE_IDENTIFIER + '~' is already a string, why call String constructor again? Just out of curiosity.